src/ol_infrastructure/substructure/aws/eks/__main__.py

```diff
@@ -668,3 +679,350 @@
         delete_before_replace=True,
     ),
 )
+
+
+############################################################
+# Install Karpenter for automatically growing and shrinking
+# the cluster
+############################################################
```

Comment:
This change seems substantial enough to merit starting to split into a module in this namespace for importing. That way changes for Karpenter can be scoped to just that module.